Adding monitoring (#547)
devonfw/hangar: documentation/logging-monitoring-kubernetes.asciidoc

> = Logging and monitoring a Kubernetes cluster

Reviewer: ```suggestion
= Logging and monitoring on Kubernetes
```

> In this section we will explain the different tools we installed to monitor Kubernetes cluster and how to access them.

Reviewer: ```suggestion
In this section we will briefly explain the different tools available for logging and monitoring on the Kubernetes cluster and how to access them.
```

== Prometheus

> You can access it with that URL: http://<hostname>/monitoring/prometheus/graph
> 
> Prometheus is an open-source monitoring solution. it collects metrics as time-series data.

Reviewer: ```suggestion
Prometheus is an open-source monitoring solution. It collects metrics from cluster and deployed applications as time-series data.

URL: `http://<ingress controller domain>/monitoring/prometheus/graph`
```

> More info: https://prometheus.io/docs/introduction/overview/

Reviewer: ```suggestion
More details on the https://prometheus.io/docs/introduction/overview/[official documentation].
```

== Loki

> (No direct access through an URL)
> 
> Loki is a solution to retrieve logs and aggregates and allows query on them. We combined this solution with `Grafana`.
> 
> More info: https://grafana.com/oss/loki/#:~:text=Loki%20is%20a%20log%20aggregation,our%20fully%20composable%20observability%20stack.

Reviewer: ```suggestion
Loki is a solution to retrieve logs from all pods and aggregates and allows query on them. You can view these logs from Grafana.

More details on the https://grafana.com/oss/loki/[official website].
```

== Grafana

> You can access it with that URL: http://<hostname>/monitoring/grafana/
> 
> Grafana is an open-source data visualization software allowing you to visualize different source of data on a same dashboard. In our cases the two data sources are `Prometheus` and `Loki`.
> 
> More info: https://grafana.com/docs/grafana/latest/introduction/

Reviewer: ```suggestion
Grafana is an open-source data visualization software allowing you to visualize different source of data on a same dashboard. In our case, the two preconfigured data sources are Prometheus and Loki.

URL: `http://<ingress controller domain>/monitoring/grafana/`

More details on the https://grafana.com/docs/grafana/latest/introduction/[official documentation].
```

== Alertmanager

> You can access it with that URL: http://<hostname>/monitoring/alertmanager/
> 
> Alertmanager handles alert received from Prometheus.
> 
> More info: https://prometheus.io/docs/alerting/latest/alertmanager/

Reviewer: ```suggestion
Alertmanager handles alert received from Prometheus.

URL: `http://<ingress controller domain>/monitoring/alertmanager/`

More details on the https://prometheus.io/docs/alerting/latest/alertmanager/[official documentation].
```
